File: backend/QuizHubApplication/DTOs/Responses/ResultDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QuizHubApplication.DTOs.Responses
{
    public record ResultDto(
     int Id,
     int QuizAttemptId,
     string QuizTitle,
     int TotalQuestions,
     int CorrectAnswers,
     int Score,
     double Percentage,
     int TimeTakenMin,
     string CreatedAt
    )
    {}
}

File: frontend/QuizHubFrontend/src/app/features/result/models/ResultDto.ts
export interface ResultDto {
    id: number;
    quizAttemptId: number;
    quizTitle: string;
    totalQuestions: number;
    correctAnswers: number;
    score: number;
    percentage: number;
    timeTakenMin: number;
    createdAt: string; // može se kastovati u Date ako želiš
}
